Sources/StreamVideoSwiftUI/CallingViews/CallingGroupView.swift

```diff
@@ -131,6 +131,7 @@ struct CircledTitleView: View {
             Text(title)
                 .foregroundColor(.white)
                 .font(fonts.title)
+                .minimumScaleFactor(0.5)
```

Comment:
Why is this needed?

Reply:
Without it the text inside the participant bubble isn't sized correctly.

| Before  | After |
| ------------- | ------------- |
| ![Simulator Screenshot - iPhone 14 Pro - 2023-08-31 at 18 08 55](https://github.com/GetStream/stream-video-swift/assets/472467/92a2db54-677e-4aa3-978b-68aa378d0ffd) | ![Simulator Screenshot - iPhone 14 Pro - 2023-08-31 at 18 10 09](https://github.com/GetStream/stream-video-swift/assets/472467/0632f39c-58ec-4c08-810c-e9e9224ba001) |

Reply:
nice fix!